### PR TITLE
Split namespace table into normative and non-normative

### PR DESCRIPTION
**Pull request by dr-shorthair**
Addressing #968

### CODE REVIEW

**Review by davebrowning — APPROVED** [review]

**Review by riccardoAlbertoni — APPROVED** [review]

File: dcat/index.html [hunk 1]
```diff
@@ -133,38 +133,55 @@ <h2>Motivation for change</h2>
...
+            <tr><td><code>time</code></td><td><code>http://www.w3.org/2006/time#</code></td></tr>
+            <tr><td><code>vcard</code></td><td><code>http://www.w3.org/2006/vcard/ns#</code></td></tr>
+            <tr><td><code>w3cgeo</code></td><td><code>http://www.w3.org/2003/01/geo/wgs84_pos#</code></td></tr>
+            <tr><td><code>xsd</code></td><td><code>http://www.w3.org/2001/XMLSchema#</code></td></tr>
+            </tbody>
+        </table>
+    
+        </section>
+        <section class="informative">
+          <h3>Namespaces appearing in examples</h3>
```
Comment (riccardoAlbertoni) [inline dcat/index.html]: @dr-shorthair : A minor thing, perhaps, I would mention that the namespaces in the second table are used also in guidelines.
If mentioning the non-normative guidelines open any issues, please just ignore my comments.

```suggestion
          <h3>Namespaces appearing in examples and non-normative guidelines</h3>
```

File: dcat/index.html [hunk 2]
```diff
@@ -133,38 +133,55 @@ <h2>Motivation for change</h2>
...
+          <p>Namespaces and prefixes used in examples in the document and not from normative parts of the recommendation are shown in the following table.</p>
```
Comment (riccardoAlbertoni) [inline dcat/index.html]:
```suggestion
          <p>Namespaces and prefixes used in examples and guidelines  in the document and not from normative parts of the recommendation are shown in the following table.</p>
```

**Comment by andrea-perego** [comment]
I wonder whether, for the headings, we should simply use:
- Normative namespaces
- Non-normative namespaces

This is the main message to be conveyed. Where those namespaces are used can be explained in the text describing the tables' content.

**Comment by agbeltran** [comment]
I agree with @andrea-perego - it seems more informative to indicate that namespaces are normative/non-normative and then explain the text where they are used.

File: dcat/index.html [hunk 3]
```diff
@@ -133,38 +133,55 @@ <h2>Motivation for change</h2>
...
-        <tr><td><code>time</code></td><td><code>http://www.w3.org/2006/time#</code></td></tr>
-        <tr><td><code>vcard</code></td><td><code>http://www.w3.org/2006/vcard/ns#</code></td></tr>
-        <tr><td><code>w3cgeo</code></td><td><code>http://www.w3.org/2003/01/geo/wgs84_pos#</code></td></tr>
-        <tr><td><code>xsd</code></td><td><code>http://www.w3.org/2001/XMLSchema#</code></td></tr>
-        </tbody>
-    </table>
+        DCAT also makes extensive use of terms from other vocabularies, in particular Dublin Core [[?DCTERMS]].
+        DCAT defines a minimal set of classes and properties of its own.</p>
+        <section>
+          <h3>Namespaces appearing in recommendations</h3>
```
Comment (riccardoAlbertoni) [inline dcat/index.html]:
```suggestion
          <h3>Normative namespaces</h3>
```

File: dcat/index.html (hunk 1 above)
Comment (riccardoAlbertoni) [inline dcat/index.html]:
```suggestion
          <h3>Non-normative namespaces</h3>
```

**Review by riccardoAlbertoni — APPROVED** [review]

File: dcat/index.html (hunk 1 above)
Comment (riccardoAlbertoni) [reply dcat/index.html]: @dr-shorthair: if you agree with  @agbeltran  and  @andrea-perego's  proposed changes, Just 
 push the commit for the above suggestions.

**Review by davebrowning — APPROVED** [review]

**Pull request merged**